### ISSUE TITLE
[Discussion] Section 2 "Extant use of Digital Object Identifiers in Archives" : Services

### ISSUE THREAD

**Issue by augustfly** [augustfly/DOI4Archives#3]
Branching discussion on "Service" DOIs from #1

**Comment by augustfly** [comment]
Originally by @gilleslandais 
  

> Services
> How to understand Service?
> 
> is it a Service ~organization including staaf and all materials (eg: VizieR service)
> eg: (organisation) vizier: https://cds.unistra.fr/registry/?verb=GetRecord&metadataPrefix=ivo_vor&identifier=ivo://cds.vizier
> (or) is it a web service ? (eg: TAPVizieR)
> eg: (web service) tapvizier: https://cds.unistra.fr/registry/?verb=GetRecord&metadataPrefix=ivo_vor&identifier=ivo://CDS.vizier/TAP
> For me, the paragraph with Collection is confusing - do you want to make the link between service and dataset ? (in that case (1) can be done, (2) is more difficult according to the metadata of Datacite)

**Comment by BaptisteCecconi** [comment]
As in #1, let's first go back to the Datacite definition of a *Service* resource: 

> **Description**: An organized system of apparatus, appliances, staff, etc., for supplying some function(s) required by end users.
> 
> **Examples and Usage Notes**: Data management service, or long-term preservation service
> 
> **Example**:  https://api.datacite.org/dois/application/vnd.datacite.datacite+xml/10.21938/3I01ISNUCODNH1ZJBCVUWA
>
> **Suggested Dublin Core Mapping**: Service


In this scope, I also would like to highlight another Datacite resource type, *InteractiveResource*, which may be of interest: 

> **Description**: A resource requiring interaction from the user to be understood, executed, or experienced.
> 
> **Examples and Usage Notes**: Training modules, files that require use of a viewer (e.g., Flash), or query/response portals
> 
> **Example**: https://api.datacite.org/dois/application/vnd.datacite.datacite+xml/10.7269/p3tb14tr
>
> **Suggested Dublin Core Mapping**: InteractiveResource

With these two definitions at hand, it seems clear that web interfaces (web portals, online tools...) belong to the *InteractiveResource*.

**Comment by gilleslandais** [comment]
the example for interactiveResource looks like a powerpoint with an aniumation.

**Comment by BaptisteCecconi** [comment]
Yes, but this is just an example. The Datacite text mentions 'query/response portals', which is typically what VO web interfaces are.
